### PR TITLE
fix set focus on input forms

### PR DESCRIPTION
**Pull request by casey1173**
Added focus to the first input element in forms with quill, which steals focus

### CODE REVIEW

File: frontend/shared/smartTags/SmartTagEditor.js [hunk 1]
```diff
@@ -5,6 +5,10 @@ import SmartTagContainer from "./SmartTagContainer";
 class SmartTagEditor extends SmartTagContainer {
     constructor($el, options) {
         $el.quillify();
+        if (options.takeFocusEl) {
```
Comment (shapiromatron) [inline frontend/shared/smartTags/SmartTagEditor.js:8]: @casey1173 I added a new option to set focus if quill steals it... let me know what you think

**Review by shapiromatron — APPROVED** [review]
Looks good. Please test out the strategy I used instead; I lot of the summary visual forms steal the focus after calling `new smartTags.SmartTagEditor`, so instead of your approach I added a new option to that constructor to change focus to a different element.

If this works and you're happy with strategy, ok to merge!

File: frontend/shared/smartTags/SmartTagEditor.js (hunk 1 above)
Comment (casey1173) [reply frontend/shared/smartTags/SmartTagEditor.js:8]: Good solution!! Glad you were able to find the source of the problem with these forms

**Comment by casey1173** [comment]
Nice solution; all summary forms are working as intended

**Pull request merged**